backends-common/cassandra/build.gradle

```diff
@@ -0,0 +1,67 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+plugins {
+    id 'java-test-fixtures'
+}
+
+configurations {
+    tests
+}
+
+dependencies {
+    implementation project(':james-core')
+    implementation project(':james-json')
+
+    implementation project(':james-server:james-server-task-api')
+    implementation project(':james-server:james-server-task-json')
+    implementation project(':james-server:james-server-util')
+
+    implementation 'com.fasterxml.jackson.core:jackson-annotations:2.9.9'
```

Comment:
If you read the comments you might have seen I spotted this, https://docs.gradle.org/current/userguide/migrating_from_maven.html#migmvn:using_boms

I'm no Gradle expert at all, but that looked like the answer to your question?

Reply:
We have several options, but let's do that later.
We can defined a property like in maven and use that.
We can use a BOM.
We can keep it like it is and do search and replace from IDE. 
I like this last one because we don't limit the module coupling. 

One option is like this: https://dev.to/jmfayard/configuring-gradle-with-gradle-properties-211k

Reply:
> We can keep it like it is and do search and replace from IDE.

Likely we would end up with diverging dependencies, auditing them will be a pain.

Reply:
None of the proposed solutions are good ones: 
* properties do not carry semantic, it's just a String and it would be a regression wrt to maven
* BOM are for release-train (when you ship a library in small pieces that should have compatible versions)
* duplication is a pain, specifically when merging stuff where it's so easy to bring an outdated version

I'm pretty sure there's a better solution, it's why I asked in the first place

Reply:
There are but IMO they are out of scope for now (like the one in the link I shared).
I would like to see the migration done and then improve it in time.
There are a lot of things we can do and in multiple ways. 
Let's take them one at a time.

Marking this as resolved, if you disagree please reopen.